economics/staking_protocol/main.tex

```diff
@@ -0,0 +1,355 @@
+\\\\
+(A) Staking Rewards provides data on the networks Tezos, Cosmos, Livepeer and Irisnet. The Staking Rewards time series is 4x daily (every 6 hours) and runs from February 2019 (Livepeer), March 2019 (Cosmos, Irisnet) and July 2019 (Tezos) until February 2020 (all four networks). Variable (1) is referred to as the \textit{adjReward}, and variable (2) as the \textit{totalStaked}. 
+\\\\
+(B) Yields provides data on the networks Tezos, Cosmos, and Livepeer. The Yields time series is daily and runs from July 2019 (Tezos \& Cosmos) and September 2019 (Livepeer) until April 2020 (all three networks). Variable (1) is referred to as the \textit{real yield}, and variable (2) as the \textit{staking rate}. 
+\\\\
+For the networks in both data sets, there are discrepancies between equivalent readings on overlapping days. This may be due to different methods of calculating both variables, but it is beyond the scope of this study to reconcile the data or decide which is superior, and so both sets of results should be interpreted with equal weight, or weighted by sample size. It is worth mentioning that these data entries have been calculated by (A) professional data providers whose products depend on data accuracy, and (B) professional staking operators that currently stake in all the networks in question. Thus both data sets certainly reflect \textbf{an} authentic economic reality, if not \textbf{the} economic reality. 
+
+\subsubsection{Relevant system dynamics}\label{dynamics}
+
+In all the networks under examination, when the staking rate increases, the real yield decreases \textbf{immediately} by a similar magnitude, and vice versa. This is because the amount of stake eligible for a subsidy increases, but the total subsidy available at that same timestamp doesn't change (or changes a relatively small amount) - this means there are fewer tokens to go around. This is a known, causal relationship, that we expect to witness when the time lag between the variables is zero. 
+\\\\
+The networks under examination have different rules for increasing and reducing the size of a stake (often referred to as `unbonding'), but typically it is asymmetric – a staker may deposit tokens more rapidly than withdraw. Livepeer stakers must trigger a withdrawal and wait about 7 days, Tezos requires a 1-2 day wait, and Cosmos \& Irisnet require 21 days. A rigorous evaluation of the impact of this asymmetry on the results is left for future work. In all the networks, a significant proportion of tokens are delegated to professional stakers. This affects the nature of staker `behavior', in particular the timing of their actions – since there may be a delay of unclear length between a delegator `undelegating' and this change being reflected in the overall staking rate. In general, we assume that changes in the real yield are experienced by delegators, and they react accordingly, if not in a precisely the same manner as stakers.
```

Comment:
in [a precisely] same manner... -> in the same manner...

Extra `a` and "precisely the same" is a bit redundant.

```suggestion
The networks under examination have different rules for increasing and reducing the size of a stake (often referred to as `unbonding'), but typically it is asymmetric – a staker may deposit tokens more rapidly than withdraw. Livepeer stakers must trigger a withdrawal and wait about 7 days, Tezos requires a 1-2 day wait, and Cosmos \& Irisnet require 21 days. A rigorous evaluation of the impact of this asymmetry on the results is left for future work. In all the networks, a significant proportion of tokens are delegated to professional stakers. This affects the nature of staker `behavior', in particular the timing of their actions – since there may be a delay of unclear length between a delegator `undelegating' and this change being reflected in the overall staking rate. In general, we assume that changes in the real yield are experienced by delegators, and they react accordingly, if not in the same manner as stakers.
```